code/modules/client/preferences.dm

```diff
@@ -105,6 +105,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	var/ambientocclusion = TRUE
 	var/auto_fit_viewport = FALSE
+	var/widescreenpref = TRUE

```

Comment:
Shouldn't this default as false?

Reply:
No, this is the default for the wide/square preference setting if it's actually enabled in the config. It being true does nothing if the two sizes are set to the exact same thing, as true and false are effectively the same thing.
I think that if you're enabling widescreen in the config, you'd want it to be the default player preference as well, no?

Reply:
Defaults in general should reflect the general use and more stable configuration, leaving this in control of the player seems the correct move.

Reply:
This IS in control of the player. They have the option to turn it off if widescreen is enabled.
I'm not seeing your point. If you're enabling widescreen on your server, why would you want players to be by default at an effective disadvantage unless they go toggle an option in a menu?

Reply:
given this is a straight upgrade and doesn't actually have disadvantages or be controversial if ever enabled, I think it being the default makes sense.

Reply:
It's absolutely controversial, and there are disadvantages.

Reply:
There aren't disadvantages for the individual client enabling it, as it currently is though I expect many things to break so it's not stable. Even if the server enables widescreen support I wouldn't want to be automatically changing a setting that's fundamentally a client side option.

Just to be clear I don't really care much about this personally, it just seems a bit wrong to be effectively changing client settings without their permission. I'm a hypocrite here though.